### PR TITLE
Adds v1 of developer guidelines.

### PR DESCRIPTION
**Pull request by gregmarra**
## Description
Adds developer guidelines, light edits to rest of page.

## Motivation and Context
Something to point to when we ask people to follow these guidelines.

## How Has This Been Tested?
Tested on local server

## Screenshots (if appropriate):
screenshot 2019-03-02 10 18 07

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would change API specifications or require data migrations)

### CODE REVIEW

File: templates/apidocs_overview.html [hunk 1]
```diff
@@ -70,13 +73,33 @@ <h2 id="webhooks">Webhooks</h2>
...
+          <ul>
+            <li>We want people to know which apps are official The Blue Alliance apps, and which apps are powered by our API, but not supported by us.</li>
+            <li>Please do not use “The Blue Alliance”, “TBA”, or The Blue Alliance lamp logo in the name or brand identity of your app.</li>
+            </li>If you’d like your project to become an official part of The Blue Alliance, please reach out to us by <a href="https://the-blue-alliance.slack.com/">joining our Slack</a>. We’d love to talk more!</li>
+          </ul>
+        </li>
+        <li>
+          <strong>Attribution</strong>
+          <ul>
+            <li>Please note that your project is “Powered by The Blue Alliance” with a link back to <a href="https://www.thebluealliance.com">thebluealliance.com</a> . This helps people discover our site, lets your users know where to report data issues, and helps more people get inspired to build on top of our APIs.</li>
```
Comment (fangeugene) [inline templates/apidocs_overview.html]: Extra space before `.`

**Comment by gregmarra** [comment]
Will fix this evening

File: templates/apidocs_overview.html [hunk 2]
```diff
@@ -70,13 +73,33 @@ <h2 id="webhooks">Webhooks</h2>
...
+
+      <h2 id="guidelines">TBA Developer Guidelines</h2>
+      <p>We’re excited to see you building things on top of The Blue Alliance’s APIs! We love seeing the creativity and utility in these projects. We have a few developer guidelines to help guide your work.</p>
+      <ol>
+        <li>
+          <strong>Branding</strong>
+          <ul>
+            <li>We want people to know which apps are official The Blue Alliance apps, and which apps are powered by our API, but not supported by us.</li>
+            <li>Please do not use “The Blue Alliance”, “TBA”, or The Blue Alliance lamp logo in the name or brand identity of your app.</li>
+            </li>If you’d like your project to become an official part of The Blue Alliance, please reach out to us by <a href="https://the-blue-alliance.slack.com/">joining our Slack</a>. We’d love to talk more!</li>
```
Comment (andrewda) [inline templates/apidocs_overview.html]:
```suggestion
            <li>If you’d like your project to become an official part of The Blue Alliance, please reach out to us by <a href="https://the-blue-alliance.slack.com/">joining our Slack</a>. We’d love to talk more!</li>
```

**Comment by gregmarra** [comment]
Thanks @andrewda for the tag fix. Also removed the space.

**Pull request merged**